fmt/testdata/FunctionCallArgsStatement/fmt.sol

```diff
@@ -0,0 +1,53 @@
+// config: bracket-spacing=true
+interface ITarget {
+    function run() external payable;
+    function veryAndVeryLongNameOfSomeRunFunction() external payable;
+}
+
+contract FunctionCallArgsStatement {
+    ITarget public target;
+
+    function estimate() public returns (uint256 gas) {
+        gas = 1 gwei;
+    }
+
+    function veryAndVeryLongNameOfSomeGasEstimateFunction()
+        public
+        returns (uint256)
+    {
```

Comment:
is this an example where @transmissions11 would want it to be `public returns (uint256) {`?

Reply:
actually no this looks good (assuming it will keep everything on the same line if i was to choose a longer max line width)

Reply:
@transmissions11 yup, if max line width is longer and it fits - then it'll stay on the same line